plugin/src/main/kotlin/com/firsttimeinforever/intellij/pdf/viewer/jcef/PdfStaticServer.kt

```diff
@@ -86,9 +87,12 @@ internal class PdfStaticServer : HttpRequestHandler() {
 
   fun getPreviewUrl(filePath: String, withReloadSalt: Boolean = false): String {
     val salt = if (withReloadSalt) Random.nextInt() else 0
-    val url = parseEncodedPath("$serverUrl/index.html?__reloadSalt=$salt&file=get-file/$filePath")
+    val url = parseEncodedPath("$serverUrl/index.html")
     val server = BuiltInServerManager.getInstance()
-    return server.addAuthToken(url).toExternalForm()
+    return server.addAuthToken(url)
```

Comment:
@PHPirates In current implementations, as there are parameters in the url, `addAuthToken` is no-op.
Referer to https://github.com/JetBrains/intellij-community/blob/master/platform/built-in-server/src/org/jetbrains/ide/BuiltInServerManagerImpl.kt#L143-L149

If `addAuthToken` should be no-op, the call of `addAuthToken` should be removed.

Reply:
Good question, I think the reason the call was here originally was because the markdown plugin also does this (https://github.com/JetBrains/intellij-community/blob/198b59116f162c7155e300650a3e46f04983d939/plugins/markdown/core/src/org/intellij/plugins/markdown/ui/preview/PreviewStaticServer.kt#L114), but I agree that with parameters it did not do anything.
I don't know if/when this is needed, but as the Markdown plugin does use `addAuthToken` I'm guessing we can leave it in.